### DISCUSSION TITLE
How to adjust the quality in the PngEncoder encoder

### DISCUSSION THREAD

**Discussion by TimurSadorov** [SixLabors/ImageSharp#2712] [Q&A] [answered]
I want to adjust the quality for the PngEncoder encoder. For example as for JpegEncoder: `new JpegEncoder() { Quality = quality }` but I can't figure out how to do it.

**Comment by antonfirsov** [comment, marked as answer]
PNG is a lossless format, there is no Quality parameter. If you want to reduce the size, you can tweak encoder settings like `CompressionLevel` or `BitDepth`.
